app/(app)/home/feed/share.tsx

```diff
@@ -16,16 +16,15 @@ import BottomSheetContainer from '../../../../components/container/BottomSheetCo
 
 export default function Share() {
   const { colors } = useTheme();
-
   return (
     <BottomSheetContainer style={[styles.main, { backgroundColor: colors.background }]}>
       <Row>
         <Row>
-          <FontAwesome name="share" size={20} color={colors.text} />
+          <MaterialCommunityIcons name="share" size={20} color={colors.text} />
           <View style={styles.gap} />
           <StyledText size="md">Share</StyledText>
         </Row>
-        <Link href="/home/feed">
+        <Link href="/home/feed/">
           <StyledText>CANCEL</StyledText>
         </Link>
```

Comment:
Clicking `CANCEL` does not close the bottom sheet.

Reply:
this is working. however I'll create a new ticket for sharing actions.

Reply:
You are right. I had some old cache lying around. I cleared the metro cache and it is working fine.